cmake/treedata/darwin_embedded/tvos/tvos.txt

```diff
@@ -1,4 +1,5 @@
-xbmc/platform/darwin/tvos             platform/tvos
-xbmc/platform/darwin/tvos/filesystem  platform/darwin/tvos/filesystem
-xbmc/platform/darwin/tvos/input       platform/darwin/tvos/input
-xbmc/windowing/tvos                   windowing/tvos
+xbmc/platform/darwin/tvos                   platform/tvos
+xbmc/platform/darwin/tvos/filesystem        platform/darwin/tvos/filesystem
+xbmc/platform/darwin/tvos/input             platform/darwin/tvos/input
+xbmc/platform/darwin/tvos/powermanagement   platform/darwin/tvos/powermanagement
+xbmc/windowing/tvos                         windowing/tvos
```

Comment:
No need to inline everything here. Just add the one powermanagement line, and leave the rest as is.